### DISCUSSION TITLE
Mqtt to serial

### DISCUSSION THREAD

**Discussion by bigbread59** [arendst/Tasmota#11138] [Q&A] [answered]
hi guys, can you help me?  Is it possible to send via mqtt strings coming from serial and vice versa using tasmota FW?  Thanks in advance

**Comment by barbudor** [comment, marked as answer]
Yes you can use SerialBridge
https://tasmota.github.io/docs/Commands/#serial-bridge

**Reply by bigbread59** [reply]
Very quick reply!!!  

Thanks a lot.
